docs/installation.rst

```diff
@@ -84,41 +82,66 @@ There are no **mandatory** external dependencies. However, some optional feature
 .. note:: You can install these additional components at any time -- before or after installing PyMuPDF. PyMuPDF will detect their presence during import or when the respective functions are being used.
 
 
-Installation from source without using an sdist
+Build and install from local PyMuPDF checkout and optional local MuPDF checkout.
 ---------------------------------------------------------
```

Comment:
This underline needs to go all the way over the parent text otherwise it won't render correctly

Reply:
I meant "under the parent text" .....

Reply:
Actually it seems to work: https://ghostscript.com/~julian/docs-PyMuPDF/installation.html

But i'll fix it anyway.

Reply:
Aha - perhaps it this was something that was just broken with a previous theme !